substrate/primitives/runtime/docs

```diff
@@ -0,0 +1 @@
+../../../docs
```

Comment:
I don't get what this file is doing?

Reply:
It think its GitHub's way of rendering links.